### ISSUE TITLE
VCF track is not shown with igv-beta.min.js

### ISSUE THREAD

**Issue by el1jaC** [igvteam/igv.js#535]
Hi all,

trying beta version of igv.js, I can't see track from vcf file; the problem doesn't occur with the latest stable release (1.0.9).
Consulting Firefox developer tools, every http get request is correct (with 200/206 return code as i expect), vcf/vcf.idx included.

Is there some piece of code in source of beta version that causes this glitch?

**Comment by jrobinso** [comment]
Could you zip up a small example that reproduces the problem?  VCF is working in general, we use it heavily.

**Comment by el1jaC** [comment]
Yes, tomorrow morning I'll send you a zip with a readme inside to reproduce the issue.

**Comment by el1jaC** [comment]
[testigvjs.zip](https://github.com/igvteam/igv.js/files/1766616/testigvjs.zip)

Open this zip with Netbeans and simply click to Run Project to launch a simple, ugly http server :)

Then, go at http://127.0.0.1:8000/index.html to see IGV web version.

In the **index.html** source code, in header section, comment lines delimited with hashtags and uncomment lines of igv beta instead.

You can see that vcf track disappears.

**Comment by jrobinso** [comment]
Yes, I can reproduce it (I don't use netbeans, but its not needed).   I will update when I find the cause, but a word of caution about the Netbeans and other IDE servers.  In general, they don't support Range headers,  so you can't use them with indexed files.

**Comment by jrobinso** [comment]
Hi, I think I found the problem.  What is the record below supposed to represent?

13	32915413	rs1234566	TTAA	.	30	PASS	NS=1;DP=1477;AF=0.4;DB;VF=252;VR=525;RF=467;RR=1010

There are no alternate alleles, and the parser doesn't know how to interpret this.  In 1.0.9 we were a little loose with this, but we are supporting a larger variety of variant types and had to tighten it up.  I think it was incorrectly interpreted as a deletion in 1.0.9. 

So if you can help me interpret this variant type I will add support for it.

**Comment by jrobinso** [comment]
I pushed a change to handle this, but I would still like help in interpreting this type of record.  What does it represent?

**Comment by el1jaC** [comment]
Hi @jrobinso ,
I've tried the change, and now everything works.

Effectively, in that record there was a deletion, but with not clear notation.

Btw, thank you (and sorry for my english :D )! We really appreciate beta version of igv web.

**Comment by jrobinso** [comment]
Hi,

If that's a deletion it is not legal VCF.   You need to add the base before
the event to both reference and alternate alleles.     See the VCF
specification.   You might have problems with other tools.

**Comment by el1jaC** [comment]
Yes I know it, but we have done a mistake with perl script that compile VCF :)
